components/frontend/src/utils.js

```diff
@@ -106,6 +117,19 @@ export function isMeasurementRequested(metric) {
     return false
 }
 
+export function isMeasurementStale(metric, reportDate) {
+    if (!metric.latest_measurement) {
+        return false
+    }
+    const end = new Date(metric.latest_measurement.end)
+    const now = reportDate ?? new Date()
+    return now - end > MILLISECONDS_PER_HOUR // No new measurement for more than one hour means something is wrong
```

Comment:
Does it not make more sense to implement such logic in the backend, where we for instance also know the collector sleep time? Picking an hour seems a bit random

Reply:
The collector doesn't know `now` though, i.e. when the user is looking at the measurement. I guess we should make the measurement frequency (`COLLECTOR_MEASUREMENT_FREQUENCY`) available to the frontend. 

By the way, we have documented that this is hardcoded in the frontend: https://quality-time.readthedocs.io/en/latest/deployment.html#configuring-measurement-frequency-optional